yammer-replace/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.arpnetworking.metrics.extras</groupId>
+    <artifactId>metrics-yammer-extra</artifactId>
+    <version>0.3.6-SNAPSHOT</version>
+  </parent>
+  <packaging>jar</packaging>
+
+  <artifactId>yammer-replace</artifactId>
+  <version>0.3.6-SNAPSHOT</version>
```

Comment:
Yes, it will.  From what I've seen, this is the recommended way.  Otherwise, apparently, the release plugin can do weird things.